apps/src/templates/PauseButton.jsx

```diff
@@ -60,27 +68,27 @@ class PauseButton extends React.Component {
 const styles = {
   icon: {
     lineHeight: 'inherit',
-    color: color.white
+    fontSize: '2.8em'
   },
   container: {
     width: 40,
     height: 40,
     lineHeight: '40px',
     textAlign: 'center',
     verticalAlign: 'middle',
```

Comment:
Do we still need all of these?

Reply:
lineHeight, yes, but not the others. Removed.